### ISSUE TITLE
ghostwriter text cursor blinks

### ISSUE THREAD

**Issue by snoermann** [wereturtle/ghostwriter#289]
I hate it when my text cursor blinks, so I always disable it. Installed ghostwriter on Linux Mint 18 Cinnamon, and see that the cursor blinks. Hope you will fix this override :-)

**Comment by appomsk** [comment]
This is not enhancement. It's about accessibility. Please either respect the operating system settings or the cursor blink settings should be in the app.
